File: 2.0/python/campaigns/postFileCampaignSound.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.campaigns.postFileCampaignSound(
    file=open('sound1.mp3', 'rb')
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/campaigns/postFileCampaignSound.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->postFileCampaignSound();
        $request->getOperationConfig()->setFileUpload(__dir__.'\sound1.mp3', 'campaign sound');
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
